doc/en/writing_plugins.rst

```diff
@@ -181,6 +181,7 @@ done via a :pep:`302` import hook which gets installed early on when
 ``pytest`` starts up and will perform this rewriting when modules get
 imported.  However since we do not want to test different bytecode
 then you will run in production this hook only rewrites test modules
+themselves (as defined by the :confval:`python_files` configuration option)
 themselves as well as any modules which are part of plugins.  Any
```

Comment:
Sorry for the misleading suggestion, now the word "themselves" appears twice!

Reply:
No worries, should have noticed myself.  How about this, also fixing the wrong "then" there:
```
diff --git i/doc/en/writing_plugins.rst w/doc/en/writing_plugins.rst
index 045e13e75..be7bd8410 100644
--- i/doc/en/writing_plugins.rst
+++ w/doc/en/writing_plugins.rst
@@ -180,11 +180,11 @@ modifies the parsed AST before it gets compiled to bytecode.  This is
 done via a :pep:`302` import hook which gets installed early on when
 ``pytest`` starts up and will perform this rewriting when modules get
 imported.  However since we do not want to test different bytecode
-then you will run in production this hook only rewrites test modules
-themselves (as defined by the :confval:`python_files` configuration option)
-themselves as well as any modules which are part of plugins.  Any
-other imported module will not be rewritten and normal assertion
-behaviour will happen.
+from what you will run in production this hook only rewrites test modules
+themselves (as defined by the :confval:`python_files` configuration option),
+and any modules which are part of plugins.
+Any other imported module will not be rewritten and normal assertion behaviour
+will happen.

 If you have assertion helpers in other modules where you would need
 assertion rewriting to be enabled you need to ask ``pytest``
```

```
imported.  However since we do not want to test different bytecode
from what you will run in production this hook only rewrites test modules
themselves (as defined by the :confval:`python_files` configuration option),
and any modules which are part of plugins.
Any other imported module will not be rewritten and normal assertion behaviour
will happen.
```

Reply:
Yes that reads well to me. I'd also add commas after the words "However" and "production".

Reply:
Ok: https://github.com/pytest-dev/pytest/pull/6741.